ui/src/timeMachine/components/view_options/OptionsSwitcher.tsx

```diff
@@ -29,6 +29,7 @@ class OptionsSwitcher extends PureComponent<Props> {
             <SingleStatOptions />
           </>
         )
+      case 'check':
```

Comment:
Similar to the comment in the view switcher. I want to make sure we separate these concepts early so that the `XYPlot` and `LineOptions` don't accumulate a bunch of branching logic later on to deal with checks.

Reply:
removed!